manager/ui/war/plugins/api-manager/html/api/api-contracts.html

```diff
@@ -25,29 +25,30 @@
+            <div class="description" apiman-i18n-key="api-contracts-help">Here is a list of all contracts between all Client Apps and this API. This provides a list of all Client Apps that may potentially consume this API.</div>
           </div>
           <div class="col-md-12" ng-show="isPublicOnly">
-            <div aria-hidden="false" style="background-color: rgb(255, 213, 185);" apiman-i18n-key="warning-public-only" data-field="onlyPublicWarning" class="alert alert-warning no-icon">Warning: this API is exclusively public, which means that it is impossible for Client Apps to create API Contracts with it.</div>
+            <div aria-hidden="false" style="background-color: rgb(255, 213, 185);" apiman-i18n-key="warning-public-only" data-field="onlyPublicWarning" class="alert alert-warning no-icon">Warning: this API is exclusively public, which means that it is impossible for Client Apps to create API contracts with it.</div>
           </div>
           <div class="col-md-12">
             <div class="table-responsive">
-              <table class="table table-striped table-bordered table-hover" data-field="contracts">
-                <thead>
+              <scrollable-table style="height: 150px">
+                <table class="table table-striped table-bordered table-hover" data-field="contracts">
+                  <thead>
                   <tr>
-                    <th width="70%" apiman-i18n-key="client-client" nowrap="nowrap">Client App</th>
+                    <th style="width: 70%" apiman-i18n-key="client-client" nowrap="nowrap">Client</th>
                     <th apiman-i18n-key="plan" nowrap="nowrap">Plan</th>
                     <th apiman-i18n-key="created-on_capitalized" nowrap="nowrap">Created On</th>
                   </tr>
-                </thead>
-                <tbody>
+                  </thead>
+                  <tbody>
 
                   <tr ng-hide="contracts.length > 0">
-                    <td colspan="4">
+                    <td colspan="3">
                       <div class="apiman-no-content container-fluid">
                         <div class="row">
                           <div class="col-md-12">
-                            <p class="apiman-no-entities-description" apiman-i18n-key="no-api-contracts">This API doesn't appear to have any contracts. This means that there aren't any client apps consuming this API.  Note: if this is a public API, then no contracts are necessary to invoke the API!</p>
+                            <p class="apiman-no-entities-description" apiman-i18n-key="no-api-contracts">This API doesn't appear to have any contracts. This means that there aren't any Client Apps consuming this API.  Note: if this is a public API, then no contracts are necessary to invoke the API!</p>

```

Comment:
I think 'Client Apps' should be lower case, and perhaps use `em` or `strong` if you want to emphasise.